docs/EffectfulHandlers.md

```diff
@@ -337,7 +337,7 @@ data from arguments.
 `-db` handlers take __one__ coeffect called `db`, and they return only __one__  effect (db again).
 
 Whereas `-fx` handlers take potentially __many__ coeffects (a map of them) and they return
-potentially __many__ effects (a map of them). So, One vs Many.
+potentially __many__ effects (a map of a vector of them). So, One vs Many.
```

Comment:
`:fx` is just yet another effect, albeit a meta one in this case. `-fx` handlers still do return a map of effects - this change is wrong.